src/test/unit/controllers/SubscriptionUrnSearchController.test.ts

```diff
@@ -0,0 +1,121 @@
+import sinon from 'sinon';
+import { Request, Response } from 'express';
+import {PipApi} from '../../../main/utils/PipApi';
+import fs from 'fs';
+import path from 'path';
+import SubscriptionUrnSearchController from '../../../main/controllers/SubscriptionUrnSearchController';
+
+const axios = require('axios');
+jest.mock('axios');
+
+
+const api = new PipApi(axios);
+const subscriptionUrnSearchController = new SubscriptionUrnSearchController(api);
+const stub = sinon.stub(api, 'getSubscriptionByUrn');
+
+describe('Subscription Urn Search Controller', () => {
+  it('should render the search page', () => {
+
+
+    const rawData = fs.readFileSync(path.resolve(__dirname, '../../../main/resources/mocks/subscriptionListResult.json'), 'utf-8');
+    const hearingsData = JSON.parse(rawData);
```

Comment:
change variable name hearingsData to something else. i.e. subscriptionResult

Reply:
yes I agree. corrected